src/apps/config/support/snabb-softwire-v1.lua

```diff
@@ -12,6 +13,51 @@ local ctable = require('lib.ctable')
 local cltable = require('lib.cltable')
 local path_mod = require('lib.yang.path')
 local generic = require('apps.config.support').generic_schema_config_support
+local binding_table = require("apps.lwaftr.binding_table")
```

Comment:
My code use to work this way but I don't know how to get an efficient effective way to search for ipv4's which fall within a certain range (look at @dpino's comment above to see the problem I hit using this method). I feel as though I'd probably just end up re-implementing what `apps.lwaftr.rangemap` implements if I tried to do it without this module.

I could always use the `rangemap` directly instead of using the code in the binding table to initialise it however I didn't really see the point in using some of the code under `apps.lwaftr` and not other parts (which do what I need them to).

WDYT?